lib/public/Util.php

```diff
@@ -489,36 +495,105 @@ public static function uploadLimit(): int|float {
 	}
 
 	/**
-	 * Get a list of characters forbidden in file names
+	 * Get a list of forbidden filename extensions that must not be used
+	 * This list should be checked case-insensitive, all names are returned lowercase.
+	 * @return string[]
```

Comment:
```suggestion
	 * @return list<string>
```
Same type change on all private vars and public getters.

Reply:
makes sense, thank you!